### ISSUE TITLE
Minor spelling issue in TicTacToe game

### ISSUE THREAD

**Issue by bhavanagundetigithub** [Shahrayar123/Python-Projects#158]
It says "sorry, you loose". The correct spelling is "lose" for a defeat in a game. Most people get this confused. I will request to solve the issue and will do a PR, my first contribution to open source.

**Comment by Ishpreet01** [comment]
@bqueenn is this issue still open?

**Comment by bqueenn** [comment]
no, i did a PR but it seems it has not seen/approved by the owners yet

**Comment by shisirkha** [comment]
same

**Comment by hitaishi-goel** [comment]
#225  this is my PR

**Comment by jayanthk82** [comment]
I wrote comments for the code.

**Comment by johnsonwa84** [comment]
Please approve my pull request for this error.

**Comment by abhi316** [comment]
is this issue still open
